lib/apiservers/service/restapi/handlers/encode/networking.go

```diff
@@ -94,20 +94,28 @@ func AsNetwork(network *executor.NetworkEndpoint) *models.Network {
 	return m
 }
 
-func AsImageFetchProxy(sessionConfig *executor.SessionConfig, http, https string) *models.VCHRegistryImageFetchProxy {
+func AsImageFetchProxy(sessionConfig *executor.SessionConfig, http, https, nproxy string) *models.VCHRegistryImageFetchProxy {
 	var httpProxy, httpsProxy strfmt.URI
+	var nProxy []strfmt.URI
 	for _, env := range sessionConfig.Cmd.Env {
 		if strings.HasPrefix(env, http+"=") {
 			httpProxy = strfmt.URI(strings.SplitN(env, "=", 2)[1])
 		}
 		if strings.HasPrefix(env, https+"=") {
 			httpsProxy = strfmt.URI(strings.SplitN(env, "=", 2)[1])
 		}
+		if strings.HasPrefix(env, nproxy+"=") {
```

Comment:
Like `config_to_data.go` (https://github.com/vmware/vic/pull/8201#discussion_r209829538), I think this would be clearer if it used a switch.

Reply:
Good catch. Done.